dev-util/rt-tests/rt-tests-1.5.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=(python3_{5,6,7})
```

Comment:
One empty line below EAPI.

Drop python3.5 support, it doesn't exist in tree anymore. If possible, test with 3.8 instead.